lib/plugin.client.js

```diff
@@ -1,4 +1,5 @@
 import VueLib from 'vue'
+import merge from 'lodash.merge'
```

Comment:
Maybe let's import it (or `klona` rather) dynamically from the place where it's needed since it won't always be needed.

Reply:
That's true, sorry for creating confusion.